configure.ac

```diff
@@ -452,6 +455,9 @@ AC_ARG_ENABLE([universalsdk],
 		UNIVERSALSDK=
 		enable_universalsdk=
 		;;
+	x86_64-*-freebsd)
```

Comment:
hang on. This diff looks weird. Maybe something messed up with applying the patch.